sbnanaobj/StandardRecord/SRTrack.cxx

```diff
@@ -8,15 +8,19 @@
 
 #include <climits>
 
+namespace{
```

Comment:
Rather than having this defined in multiple places, would it be preferable to have it defined somewhere common? (e.g. SREnums)

Reply:
I think SREnums is the wrong place, but I basically agree. Perhaps this ties in with the comment elsewhere about where `-5` should be defined.

Reply:
I filed https://github.com/SBNSoftware/sbnanaobj/issues/24